src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3465,7 +3460,7 @@ internal EffectiveParameters(ImmutableArray<TypeWithAnnotations> types, Immutabl
                                             ref useSiteInfo);
                         if (typeArguments.IsDefault)
                         {
-                            return new MemberResolutionResult<TMember>(member, leastOverriddenMember, inferenceError);
+                            return new MemberResolutionResult<TMember>(member, leastOverriddenMember, inferenceError, hasTypeArgumentInferredFromFunctionType: false);
```

Comment:
>hasTypeArgumentInferredFromFunctionType: false [](http://example.com/codeflow?start=118&length=46)

Why not simply pass ```hasTypeArgumentsInferredFromFunctionType```?

Reply:
We could although when `typeArguments.IsDefault`, then `hasTypeArgumentsInferredFromFunctionType == false`.